man/profile_emissions_upstream.Rd

```diff
@@ -53,7 +53,7 @@ profile products.}
 \item{ecoinvent_inputs}{Dataframe. Upstream products from ecoinvent.}
 }
 \value{
-A data frame with the column \code{companies_id}, and the nested columns\code{product} and \code{company} holding the outputs at product and company level. Unnesting \code{product} yields a data frame with at least columns \code{companies_id}, \code{grouped_by}, \code{risk_category}. Unnesting \code{company} yields a data frame with at least columns \code{companies_id}, \code{grouped_by}, \code{risk_category}, \code{value}.
+A data frame with the column \code{companies_id}, and the list columns \code{product} and \code{company} holding the outputs at product and company level.
```

Comment:
I'm removing everything except the first sentence. The removed documentation is correct for tiltIndicator but not for tiltIndicatorAfter.